### PR TITLE
Fix: Excessive DOM output when using grid container [ED-13288]

### PR DESCRIPTION
**Pull request by louiswol94**
Before:

`<div class="elementor-element elementor-element-a1ba666 e-grid e-con-boxed e-con e-parent" data-id="a1ba666" data-element_type="container" data-settings="{&quot;container_type&quot;:&quot;grid&quot;,&quot;grid_columns_grid_tablet&quot;:{&quot;unit&quot;:&quot;custom&quot;,&quot;size&quot;:&quot;&quot;,&quot;sizes&quot;:[]},&quot;grid_columns_grid_mobile&quot;:{&quot;unit&quot;:&quot;custom&quot;,&quot;size&quot;:1,&quot;sizes&quot;:[]},&quot;grid_rows_grid_tablet&quot;:{&quot;unit&quot;:&quot;custom&quot;,&quot;size&quot;:&quot;&quot;,&quot;sizes&quot;:[]},&quot;grid_rows_grid_mobile&quot;:{&quot;unit&quot;:&quot;custom&quot;,&quot;size&quot;:&quot;&quot;,&quot;sizes&quot;:[]},&quot;content_width&quot;:&quot;boxed&quot;,&quot;grid_outline&quot;:&quot;yes&quot;,&quot;grid_columns_grid&quot;:{&quot;unit&quot;:&quot;fr&quot;,&quot;size&quot;:3,&quot;sizes&quot;:[]},&quot;grid_rows_grid&quot;:{&quot;unit&quot;:&quot;fr&quot;,&quot;size&quot;:2,&quot;sizes&quot;:[]},&quot;grid_auto_flow&quot;:&quot;row&quot;,&quot;grid_auto_flow_tablet&quot;:&quot;row&quot;,&quot;grid_auto_flow_mobile&quot;:&quot;row&quot;}" data-core-v316-plus="true">`

After:

`<div class="elementor-element elementor-element-a1ba666 e-grid e-con-boxed e-con e-parent" data-id="a1ba666" data-element_type="container" data-settings="{&quot;container_type&quot;:&quot;grid&quot;,&quot;content_width&quot;:&quot;boxed&quot;}" data-core-v316-plus="true">`

### CODE REVIEW

File: includes/controls/groups/grid-container.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ protected function init_fields() {
 			'label_on' => esc_html__( 'Show', 'elementor' ),
 			'label_off' => esc_html__( 'Hide', 'elementor' ),
 			'default' => 'yes',
-			'frontend_available' => true,
+			'editor_available' => true,
```
Comment (louiswol94) [inline includes/controls/groups/grid-container.php:36]: Changing the key to `editor_available` will make sure the settings will not render on the FE of the website, but still be available in the Editor.

**Review by Barcov — DISMISSED** [review]
@louiswol94 Nice

**Review by hein-obox — DISMISSED** [review]

File: assets/dev/js/frontend/handlers/base.js [hunk 2]
```diff
@@ -197,7 +197,7 @@ module.exports = elementorModules.ViewModule.extend( {
 				settingsKeys = elementorFrontend.config.elements.keys[ type ] = [];
 
 				jQuery.each( settings.controls, ( name, control ) => {
-					if ( control.frontend_available ) {
+					if ( control.frontend_available || control.editor_available ) {
```
Comment (KingYes) [inline assets/dev/js/frontend/handlers/base.js:200]: Add the new property to this unset code: https://github.com/elementor/elementor/blob/6980d7cc46431f4554bf5b60fef181434f277cca/includes/base/controls-stack.php#L434C15-L451

Comment (mykytamurzin) [reply assets/dev/js/frontend/handlers/base.js:200]: @KingYes Done

**Comment by rami-elementor** [comment]
I love this fix!

**Comment by louiswol94** [comment]
> I love this fix!

Thanks @rami-elementor :)

**Review by murzin-nikita — APPROVED** [review]

**Pull request merged**